components/cosmetic_filters/resources/data/element_picker.ts

```diff
@@ -685,9 +687,12 @@ const launchElementPicker = (root: ShadowRoot) => {
     quitElementPicker()
   })
 
-  const manageButton = root.getElementById('btnManage')!
-  manageButton.addEventListener('click', () => {
-    api.cosmeticFilterManage();
+  const resetButton = root.getElementById('btnReset')!
+  resetButton.addEventListener('click', () => {
+    api.cosmeticFilterReset(() => {
+      quitElementPicker()
+      location.reload()
```

Comment:
Is it possible to refresh the page from C++ handler? If yes we could remove `promise` and related code.